packages/core-snapshots-cli/lib/db/queries/index.js

```diff
@@ -0,0 +1,29 @@
+const loadQueryFile = require('../utils/load-query-file')
+
+module.exports = {
+  blocks: {
+    heightRange: loadQueryFile(__dirname, './blocks/height-range.sql'),
+    latest: loadQueryFile(__dirname, './blocks/latest.sql'),
+    findByHeight: loadQueryFile(__dirname, './blocks/find-by-height.sql'),
+    deleteFromHeight: loadQueryFile(__dirname, './blocks/delete-from-height.sql')
+  },
+  transactions: {
+    timestampRange: loadQueryFile(__dirname, './transactions/timestamp-range.sql'),
+    deleteFromTimestamp: loadQueryFile(__dirname, './transactions/delete-from-timestamp.sql')
+  },
+  rounds: {
+    deleteFromRound: loadQueryFile(__dirname, './rounds/delete-from-round.sql')
+  },
+  transactionsExport: (start, end) => {
+    return `SELECT id, block_id, version, sequence, timestamp, sender_public_key, recipient_id, type, vendor_field_hex, amount, fee, serialized from TRANSACTIONS WHERE TIMESTAMP BETWEEN ${start} AND ${end} ORDER BY TIMESTAMP`
+  },
+  blocksExport: (start, end) => {
```

Comment:
All SQL queries should be in files and be loaded via `loadQueryFile` to make sure we get all benefits of https://vitaly-t.github.io/pg-promise/QueryFile.html.